specification/dimensions/tags.md

```diff
@@ -0,0 +1,76 @@
+# Tags
+
+The Tags column represents the set of finalized user-defined and/or provider-defined tags assigned to Tag Sources.  Tags are commonly used for scenarios like adding business context to billing data to identify and accurately allocate charges.
+
+A tag is 'finalized' after a single tag value is chosen from a set of corresponding tag values for a single tag key.  Whether a tag key can have multiple, possible values is determined by the Provider.
+
+The Tags column adheres to the following requirements:
+
+* The Tags column MUST contain user-defined and/or provider-defined tags.
+* The Tags column MUST only contain finalized tags.
+* The Tags column MUST be in Key-Value Format.
+* A Tag without a specified value MUST have its value set to null.
+* Providers MUST NOT alter user-defined Tag keys or values.
+
+Provider-defined Tags additionally adhere to the following requirements:
+
+* Provider-defined tags MUST be prefixed with a provider-specified tag key prefix.
+* Providers SHOULD publish all provider-specified tag key prefixes within their respective documentation.
+
+## Provider-Defined vs. User-Defined Tags
+
+The following is an example of one user-defined tag and one provider-defined tag, respectively, with tag key, `foo`.  The first tag, which is user-defined, is not prefixed. The second tag is prefixed with marketplace/ which the provider has specified as a reserved tag key prefix.
+
+```json
+    {
+        "foo":"bar",
+        "marketplace/foo": "bar"
+    }
+```
+
+## Finalized Tags
+
+A Tag can either be static or dynamic. If a Tag is static, its value is immutable. If a Tag is dynamic, its value is determined by a set of predefined user or provider rules. A finalized Tag is the final result of any static or dynamic Tag.
+
+As a example, let's assume 1 sub account exists with 1 virtual machine with the following details:
+
+* Sub account
+  * id: *my-account*
+  * user-defined tags: *team:ops*, *env:prod*
+* Virtual machine
+  * id: *my-vm*
+  * user-defined tags: *team:web*
+
+The table below represents a finalized cost and usage dataset with these resources.  It also shows the finalized state after all resource-oriented, tag inheritance rules are processed.
+
+| ResourceType    | ResourceId | Tags                                        |
+| :---------------| :----------| :-------------------------------------------|
+| Account         | my-account | { "team": "ops", "env": "prod" }            |
+| Virtual Machine | my-vm      | { "team": "web", *"env": "prod"* }          |
+
+Because the the Virtual Machine Resource did not have an `env` tag, it inherited tag, `env:prod` (italicized), from its parent Account.  Conversely, because the Virtual Machine Resource already has a `team` tag (`team:web`), it did not inherit `team:ops` from its parent Account.
+
+## Column ID
+
+Tags
+
+## Display Name
+
+Tags
+
+## Description
+
+The set of finalized tags assigned to Tag Sources.
```

Comment:
Yeah I'm pro removing mentions of the following terms:

* static
* dynamic
* finalized

Put it another way, what is the change we want to effect, resulting attribute of data we are trying to gain, or problem we want to solve by taxonomizing tags in this fashion?

Reply:
I misread this earlier. @joshk0 @flanakin What's the issue with finalized? We want to explain that this is the reduced set of values once inheritance rules have been taken into consideration.

Static/dynamic I agree with and have felt should be described differently as well.

Reply:
Leaving current text as is and we can revist post-1.0

Reply:
I agree with Michael that "finalized" in a short-form definition here is problematic because it's not a Googleable term.  Perhaps a link to the glossary with an entry for "finalized" could help solve that.  Don't delay on resolution on my part -- but I did want to weigh in.

Reply:
I believe static/dynamic/finalized are not concepts that are common to all providers.
Every provider has its own tag processing and FOCUS just needs to specify the format for the resulting tags without any assumption on the upstream processing.

Reply:
Agree with Shawn/Michael.  Will add `Finalized Tag` to the glossary